src/jit-diff/jit-diff.cs

```diff
@@ -551,15 +551,24 @@ public static int Main(string[] args)
             return ret;
         }
 
-        public static CommandResult TryCommand (string name, IEnumerable<string> commandArgs)
+        public static CommandResult TryCommand (string name, IEnumerable<string> commandArgs, bool capture = false)
         {
             try 
             {
                 Command command =  Command.Create(name, commandArgs);
 
-                // Wireup stdout/stderr so we can see outout.
-                command.ForwardStdOut();
-                command.ForwardStdErr();
+                if (capture)
```

Comment:
Is this diff related to the other changes?  If not, should you at least comment on it in the description?